config/cookstyle.yml

```diff
@@ -135,6 +185,10 @@ Chef/WhyRunSupportedTrue:
   Description: why_run_supported? no longer needs to be set to true as it is the default in Chef 13+
   Enabled: true
   VersionAdded: '5.1.0'
+    # Include:
```

Comment:
Is there a reason these are added commented out?

Reply:
I need to further test them before I make them the default, but I want to remember that we should probably path limit these to speed up cookstyle